crates/cli/src/subcommands/project/rust/Cargo._toml

```diff
@@ -9,5 +9,5 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-spacetimedb = "1.0.0-rc3"
+spacetimedb = "1.0.0-rc3.1"
```

Comment:
Given that these changes have been released, should we be bumping to a later version? (Or, if API/ABI-breaking changes have merged on master since `rc3` was released, should we be bumping to `rc4` here?)